tensorflow_addons/seq2seq/tests/basic_decoder_test.py

```diff
@@ -24,372 +25,301 @@
 from tensorflow_addons.seq2seq import sampler as sampler_py
 
 
-class BasicDecoderTest(tf.test.TestCase, parameterized.TestCase):
-    """Unit test for basic_decoder.BasicDecoder."""
-
-    @parameterized.named_parameters(
-        ("use_output_layer", True), ("without_output_layer", False)
+def test_step_with_training_helper_output_layer(use_output_layer=True):
```

Comment:
```suggestion
@pytest.mark.parametrize("use_output_layer", [True, False])
def test_step_with_training_helper_output_layer(use_output_layer):
```

Reply:
Oh, yeah, it also tests none use_output_layer. Thanks @gabrieldemarmiesse